docs/pages/avatar.js

```diff
@@ -69,19 +256,21 @@ export default function AvatarPage({ generatedDocGen }: {| generatedDocGen: DocG
   />
 </Flex>
 `}
-      />
-
-      <Example
-        description={`
-    Avatars without a \`size\` prop will be expand to fit to the width of their
+          />
+        </MainSection.Subsection>
+        <MainSection.Subsection
+          description={`
+    Avatars without a \`size\` prop will expand to fit the width of their
     parent container. A common use case is to achieve column-based sizing.
 
     Resize the browser to see these Avatar change to match the width of the \`Column\` they
```

Comment:
Can we fix this? 
![image](https://user-images.githubusercontent.com/10593890/142044230-5d052040-2004-4221-83d6-92f7d3722384.png)
 @ayeshakmaz